db/db_impl/db_impl.cc

```diff
@@ -6380,7 +6380,7 @@ void DBImpl::RecordSeqnoToTimeMapping() {
   bool appended = false;
```

Comment:
Very minor. According to this:
```
// it is assumed that the recorded time in each pair
// comes at or after the sequence number and before the next sequence number,
```
And the above documentation, I think above two steps should be swapped in order to be consistent with the documentation?

Reply:
Thanks for catching that!

(If we standardized on sampling the time right before the seqno, we would technically need to make GetProximalSeqnoBeforeTime peer back one step further, making the results less accurate.)